src/app/Table.elm

```diff
@@ -1,29 +1,91 @@
-module Table exposing (view, init, Msg)
+module Table exposing (view, init, initWithPlayersAndCup)
+
+{-|
+
+   This module handles the table of players,
+   and everything relating to that.
+
+-}
 
 import Html exposing (..)
+import Html.Events exposing (..)
+import Types exposing (PlayerID)
+import TableMsg exposing (Msg(..))
+import Dice
 
 
-{-
+type alias PlayerName =
+    String
 
-   This module handles the view rendering for the table of players
-   and everything relating to that.
 
--}
+type alias DiceCount =
+    Int
+
 
+type alias TablePlayer =
+    { id : PlayerID
+    , name : PlayerName
+    , diceCount : DiceCount
+    }
 
-type Model
-    = Model
 
+type alias TablePlayers =
+    List TablePlayer
 
-type Msg
-    = Msg
+
+type alias Model =
```

Comment:
What reason? I find it much worse. `Table.Table` ?! My definition of `Model` is the data that a module uses to model the data it's concerned with — to model the world. I like the name `Model` because of this.

Why would you prefer to call it `Table` ?